app/view_models/admin/auction_show_view_model.rb

```diff
@@ -82,10 +86,6 @@ def html_summary
     MarkdownRender.new(auction.summary).to_s
   end
 
-  def admin_edit_auction_partial
```

Comment:
we do, I just inlined this.  (` <%= render partial: 'auctions/edit_auction_link', locals: { view_model: @view_model } %>` ) I think this method was leftover from a world where we shared view models between admin and regular auction  show pages.